generic/piranha-tree-sitter/src/cleanup_rules/java/rules.toml

```diff
@@ -0,0 +1,484 @@
+# The language specific rules in this file are applied after the API spcific change has been performed.
+
+# Before : 
+#  if (true) { doSomething(); }
+# After :
+#  { doSomething(); } 
+# 
+# Before : 
+#  if (true) { doSomething(); } else { doSomethingElse();}
+# After :
+#  { doSomething(); } 
+#
+[[rules]]
+name = "If statement - true"
+query = """
+(
+    (if_statement
+        condition : ((parenthesized_expression ((true) @i)))
+        consequence : ((statement) @consequence))
+@if_statement)
+"""
+replace_node = "if_statement"
+replace = "@consequence"
+groups = ["If cleanup"]
+
+# Before : 
+#  if (true) { doSomething(); } else { doSomethingElse();}
+# After :
+#  { doSomethingElse(); } 
+#
+# Before : 
+#  if (true) { doSomething(); }
+# After :
+#  
+[[rules]]
+name = "If statement - false"
+query = """
+(
+    (if_statement
+        condition : ((parenthesized_expression ((false) @i)))
+        consequence : ((statement) @consequence)
+        alternative : ((_) @alternative) ?)
+@if_statement)"""
+replace_node = "if_statement"
+replace = "@alternative"
+groups = ["If cleanup"]
+
+# Before : 
+#  !false
+# After :
+#  true 
+#
+[[rules]]
+name = "not false"
+query = """
+(
+    (unary_expression
+        operator: "!"
+        operand: (false))
+@unary_expression)
+"""
+replace_node = "unary_expression"
+replace = "true"
+groups = ["Boolean expression simplify"]
+
+
+# Before : 
+#  !true
+# After :
+#  false
+#
+[[rules]]
+name = "not True"
+query = """
+(
+    (unary_expression
+        operator: \"!\"
+        operand: (true))
+@u)
+"""
+replace_node = "u"
```

Comment:
why `u`? instead of `@unary_expression`? 
Though this clarifies that it is the reference to line 79 and not to 76. If so, worthwhile to update others to use similar variable names (e.g., line 60)